src/main/java/org/spongepowered/api/command/CommandMessageFormatting.java

```diff
@@ -32,12 +32,18 @@
     private CommandMessageFormatting() {
     }
 
+    public static final Text FORWARD_SLASH = Text.of("/");
```

Comment:
These are only Text constants that are only here because the Exceptions are in the API.

We could make a catalog type out of them, but I would then just suggest we go through the API and catalog all the text constants into a `TextConstant`/`TextConstants` catalog, so that we only have one of each constant. We might just want to gather all API exposed Text constants together in one class though.